### PR TITLE
list_of_events: Add indexer_worker_indexing_finished event

### PR DESCRIPTION
**Pull request by siavasht**

### CODE REVIEW

File: source/admin_manual/list_of_events.rst [hunk 1]
```diff
@@ -1067,6 +1067,31 @@ A mail index file was recreated.
...
+| first_uid           | UID of the first indexed message                       |
++---------------------+--------------------------------------------------------+
+| last_uid            | UID of the last indexed message                        |
++---------------------+--------------------------------------------------------+
+| user_cpu_usecs      | Total user mode cpu time of the process in             |
+|                     | microseconds at the time of sending event.             |
++---------------------+--------------------------------------------------------+
+
+
+.. _event_indexer_worker_indexing_finished:
```
Comment (sirainen) [inline source/admin_manual/list_of_events.rst:1091]: This should be before the title. And .. looks like the existing event_mail_index_recreated is also in the wrong place, so should fix that as well.

File: source/admin_manual/list_of_events.rst [hunk 2]
```diff
@@ -1067,6 +1067,31 @@ A mail index file was recreated.
...
+| Field               | Description                                            |
++=====================+========================================================+
+| message_count       | Number of messages indexed                             |
++---------------------+--------------------------------------------------------+
+| first_uid           | UID of the first indexed message                       |
++---------------------+--------------------------------------------------------+
+| last_uid            | UID of the last indexed message                        |
++---------------------+--------------------------------------------------------+
+| user_cpu_usecs      | Total user mode cpu time of the process in             |
+|                     | microseconds at the time of sending event.             |
```
Comment (sirainen) [inline source/admin_manual/list_of_events.rst:1087]: Total user CPU spent on the indexing transaction.

(It looks like "user CPU" is more common than "user mode CPU")

**Review by sirainen — CHANGES_REQUESTED** [review]

File: source/admin_manual/list_of_events.rst (hunk 1 above)
Comment (siavasht) [reply source/admin_manual/list_of_events.rst]: Indeed. Both should be fixed now.

File: source/admin_manual/list_of_events.rst (hunk 2 above)
Comment (siavasht) [reply source/admin_manual/list_of_events.rst]: Reworded.

File: source/admin_manual/list_of_events.rst [hunk 3]
```diff
@@ -1064,7 +1067,27 @@ A mail index file was recreated.
...
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. versionadded:: 2.3.15
+
+Indexer worker process completed an indexing transaction.
+
++---------------------+--------------------------------------------------------+
+| Field               | Description                                            |
++=====================+========================================================+
+| message_count       | Number of messages indexed                             |
```
Comment (sirainen) [inline source/admin_manual/list_of_events.rst:1082]: Link to parent event? Probably mailbox?

**Review by sirainen — APPROVED** [review]

**Pull request merged**